events/migrations/0055_fix_chinese_language_code.py

```diff
@@ -0,0 +1,148 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.11 on 2018-05-30 05:48
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def fix_chinese_language_object(apps, schema_editor):
+    Language = apps.get_model('events', 'Language')
+
+    try:
+        chinese_lang = Language.objects.get(id="zh")
+        chinese_lang.id = "zh_hans"
+        chinese_lang.save()
```

Comment:
Also, what happens here is that at least for me, the django ORM creates a copy of Chinese with the new primary key. The original object remains, and all foreign key references still point to the old `zh` primary key and not the new one :(

Reply:
Also, for some reason, both `zh` and `zh_hans` have `translation_available` as false, even though `ar` and `ru` list it as true :(

Reply:
Probably something with how modeltranslate serializes the `translation_available` field.

Reply:
I fixed 'def get_translation_available()' last time I ammended my commit. So, translation available shows `True` for me.
I will fix the foreign key links and extra copies and all that ORM jazz.